.circleci/config.yml

```diff
@@ -46,16 +61,17 @@ commands:
           name: Gather test results
           when: always
           command: |
+            rm -rf build/test-output
             FILES=`find . -name test-results`
             for FILE in $FILES
             do
               MODULE=`echo "$FILE" | sed -e 's@./\(.*\)/build/test-results@\1@'`
-              TARGET="build/test-results/$MODULE"
+              TARGET="build/test-output/$MODULE"
```

Comment:
q: why the rename?

Reply:
this was conflicting with test-results which is also created by gradle when it run tests. Hence I rename it so that my remove command doesn't remove test-results.